### DISCUSSION TITLE
Create swipe-through intro screens

### DISCUSSION THREAD

**Discussion by okdistribute** [digidem/mapeo-mobile#821] [Ideas]
"- Use the ""Intro"" designs from https://www.notion.so/digidem/MM-Beta-UX-Refined-UIs-56504c2e739d426d81201e7fb9b116e9
- Also implement the ""Enable GPS (intro)"" screen that asks the user for permission to use GPS.
- Should use react-reanimated (or a module that uses it) for good performance on old devices (I think https://github.com/react-native-community/react-native-tab-view is the best but maybe for react-navigation integration https://github.com/react-navigation/tabs)
- Should integrate with react-navigation, show only on first startup, and not leave screens in the ""Stack"" e.g. after viewing them the back button does not return to them, but during viewing them the back button will move back through them)."

**Comment by gmaclennan** [comment]
This is implemented in the WCMC variant through [wcmc-mapeo-mobile-intro-screens](https://github.com/digidem/wcmc-mapeo-mobile-intro-screens). Implementing this for the main variant should be just a case of forking this module and adapting the text and images.

**Comment by gmaclennan** [comment]
Created a placeholder module for the intro screens: https://github.com/digidem/mapeo-mobile-intro
